bitbots_hcm/bitbots_hcm/hcm_dsd/hcm.dsd

```diff
@@ -1,48 +1,37 @@
-#ShutDownProcedure
-$PickedUp
-    PICKED_UP --> @TurnMotorsOff, @StayShutDown
-    ON_GROUND --> @PlayAnimationSitDown, @TurnMotorsOff, @StayShutDown
-
 -->HCM
 $StartHCM
-    NOT_WALKREADY --> @Wait + time:0.1 + r:false, @PlayAnimationDynup + direction:walkready + initial:true + r:false
-    SHUTDOWN_REQUESTED --> #ShutDownProcedure
-    SHUTDOWN_WHILE_HARDWARE_PROBLEM --> @StayShutDown
+    START_UP --> @RobotStateStartup, @Wait + time:0.1 + r:false, @PlayAnimationDynup + direction:walkready, @Wait
     RUNNING --> $Stop
-        STOPPED -->  @ForceStopWalking, @PlayAnimationDynup + direction:walkready + initial:true + r:false, @StayStopped
-        FREE -->$Record
-            RECORD_ACTIVE --> @StayRecord
+        STOPPED --> @CancelGoals, @StopWalking, @PlayAnimationDynup + direction:walkready, @Wait
+        FREE -->$RecordAnimation
+            RECORD_ACTIVE --> @RobotStateRecord, @Wait
             FREE --> $CheckMotors
-                MOTORS_NOT_STARTED --> @WaitForMotorStartup
-                OVERLOAD --> @PlayAnimationDynup + direction:descend, @TurnMotorsOff, @StayMotorsOff
-                PROBLEM --> @WaitForMotors
-                TURN_OFF --> @PlayAnimationDynup + direction:descend, @TurnMotorsOff, @StayMotorsOff
-                TURN_ON --> @TurnMotorsOn, @PlayAnimationDynup + direction:rise
+                MOTORS_NOT_STARTED --> @RobotStateStartup, @WaitForMotorStartup
+                OVERLOAD --> @RobotStateMotorOff, @CancelGoals, @StopWalking, @PlayAnimationFallingFront, @TurnMotorsOff, @Wait
```

Comment:
Why did you change it to play falling front animation when we are having overload errors?

Reply:
Because sitting down might not be desirable if we have an overload and springs in the knees. The falling front animation will result in the robot falling to the front in a more or less controlled manner, which is probably better.